README.md

```
@@ -1,28 +1,29 @@
 
 # Fuelrod docker compose
 
-Docker compose tool for combining traefik reverse proxy with docker containers to allow domain name serving
+Docker compose tool for combining reverse proxy with docker containers to allow domain name serving
 
 [NGINX Rate Limiting](https://www.nginx.com/blog/rate-limiting-nginx/#:~:text=Rate%20%E2%80%93%20Sets%20the%20maximum%20request,1%20request%20every%20100%20milliseconds)
 
-## Certbot 
+### Certbot 
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)